isis/src/base/apps/ckwriter/CkSpiceSegment.cpp

```diff
@@ -265,9 +267,21 @@ void CkSpiceSegment::import(Cube &cube, const QString &tblname) {
     _utcStartTime = toUTC(startTime());
     _utcEndTime   = toUTC(endTime());
 
-    _timeOffset =  fabs(etLabStart.Et() - startTime());
+    _startOffset =  etLabStart.Et() - startTime();
+    _endOffset =  etLabEnd.Et() - endTime();
     // account for padding
-    if (_timeOffset <= 0.003) { _timeOffset = 0; }
+    if (_startOffset >= 0.0031) {
```

Comment:
If the padding is .003, should this just be > .003 instead of >=.0031?

Reply:
@AustinSanders  I was running into a situation in which the offset was 0.00305282 I guess due to the precision of doubles? Would a better solution be to set the precision?

Reply:
Hmm.  I'm not really sure.  With the current solution, what happens if the offset falls between .003 and .0031?

Reply:
Fixed this check by first rounding offsets to three decimal places before doing the padding check. It is necessary that these offsets are within three decimals anyways for when we add/subtract the label time with these offsets.